users/forms.py

```diff
@@ -0,0 +1,69 @@
+"""
+Users application forms
+"""
+
+from django import forms
+from django.contrib.auth import authenticate
+from django.core.exceptions import ValidationError
+from users.models import UserModel
+
+
+class RegisterForm(forms.Form):
+    """Form for registering new user"""
+    username = forms.CharField(max_length=64, label="Ім'я користувача")
+    email = forms.EmailField(label="email")
+    password1 = forms.CharField(
+        max_length=255, label="Пароль", widget=forms.PasswordInput)
```

Comment:
У такому разі - роби перенос закриваючої дужки - PEP8
```suggestion
        max_length=255, label="Пароль", widget=forms.PasswordInput
    )
```